runtime/stmt.h

```diff
@@ -245,12 +245,13 @@ class clone_buffer : public stmt_node<clone_buffer> {
 class crop_buffer : public stmt_node<crop_buffer> {
 public:
   var sym;
```

Comment:
I think it's a bit confusing: ```sym```  before was both src and dst, but now it's more like dst, so I think this should be explained in a comment at least.

Reply:
Fixed stale comments

Reply:
BTW, I'm trying to keep all of these "declaring" ops using `sym` as the declared object. We have template code that handles all of them, which assumes the name `sym` in this case.